conda/environments/all_cuda-118_arch-x86_64.yaml

```diff
@@ -63,7 +63,7 @@ dependencies:
 - rmm==23.6.*
 - scikit-build>=0.13.1,<0.17.2
 - scikit-learn==1.2
-- scipy
+- scipy>=1.8.0
```

Comment:
Version 1.8.0 or higher contains a deprecation warning telling users to use the namespace `scipy.optimize` instead of `scipy.optimize.optimize`. It might be possible to use `scipy.optimize` in earlier versions but I know this is a safe lower bound.